junifer/markers/parcel_aggregation.py

```diff
@@ -184,9 +184,9 @@ def compute(
             img=parcellation_img_res,
         )
 
-        if self.mask is not None:
-            logger.debug(f"Masking with {self.mask}")
-            mask_img = get_mask(mask=self.mask, target_data=input)
+        if self.masks is not None:
+            logger.debug(f"Masking with {self.masks}")
+            mask_img = get_mask(masks=self.masks, target_data=input)
```

Comment:
this needs to hand over the `extra_input` to get_mask

Reply:
good catch!